nilearn/_utils/testing.py

```diff
@@ -10,6 +10,8 @@
 import sys
 import tempfile
 import warnings
+import resource
```

Comment:
> It is for Unix systems. I probably should skip tests if not available /
> setrlimit or getrlimit raise a resource.error.

More than skip tests, you cannot import it at all windows, I suspect.
Which means that importing scikit-learn on windows would crash.

We need CI for Windows. I created an issue: #804.